### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
Home page for e-com app.

Site link: https://ym-store-homepage.netlify.app/

### CODE REVIEW

**Comment by varun-Tad** [comment]
Outline is appearing when you type something in the search input box. remove the outline

File: index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            </a>
+        </div>
+    </header>
+
+    <main>
+        <!-- Hero Section Starts -->
+        <section id="hero">
+            <div class="hero-wrapper">
+                <div class="hero-text-container">
+                    <h1 style="color:var(--color-green-600);">YOGA ACCESSORIES</h1>
```
Comment (varun-Tad) [inline index.html]: Avoid inline styling

File: home.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    margin: 0 auto;
+}
+
+.m-bottom {
+    margin-bottom: 0.5rem;
+}
+
+/* Hero Section */
+#hero {
+    background-color: var(--color-green-100);
```
Comment (nikhiljugale007) [inline home.css:27]: avoid using id for styling.

**Review by nikhiljugale007 — COMMENTED** [review]
One UI suggestion
In  header you have added icon for cart and also text "CART" not sure why you did this.

**Pull request merged**